lib/ansible/modules/cloud/openstack/os_project.py

```diff
@@ -99,7 +99,10 @@
 '''
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.openstack import openstack_full_argument_spec, openstack_module_kwargs, openstack_cloud_from_module
+from ansible.module_utils.openstack import (
```

Comment:
While this works I'm not a fan of multiple imports on a single line. Not something I'd be willing to block the review over but something I'd suggest breaking out into multiple lines.